Updated the cloud providers.

vdbhb59 opened the pull request:
Removed:
1. vern instance as is dead for sometime now, and the whole vern domain itself is dead.
2. Facil services, as they no longer provide for FreshRSS instance.
3. Zaclys instance as they too no longer provide RSS instance.

Updated the RSS direct instance details and URLs, so that they take users to the correct place instead of having to find, navigate, locate and what not in the pages of the domains they are hosted in. Makes it much simpler and directs to path of the instance itself.

Alkarex commented on pages/cloud-providers.html:
```diff
@@ -28,33 +28,18 @@
 
             <p>
-                <strong><a href="https://cheredeprince.net/">cheredeprince.net</a></strong>
+                <strong><a href="https://rss.cheredeprince.net/">cheredeprince.net</a></strong>
```
I think we should keep the landing page, and potentially also indicate the direct link. Otherwise there is the reverse problem, namely that the user lands on a FreshRSS instance without knowing who operates it, why, the conditions, etc.

marienfressinaud replied:
I agree, my intention was to conduct people to the global presentation of each provider so they could understand where they're going and who is going to manage their data. It helps to not consider the providers as some impersonal structure.

vdbhb59 replied:
> I think we should keep the landing page, and potentially also indicate the direct link. Otherwise there is the reverse problem, namely that the user lands on a FreshRSS instance without knowing who operates it, why, the conditions, etc.

How does this look for a landing page? Within the card-info itself:
`        <div class="card flow-small">
            <div class="card__title">
                <strong><a href="https://rss.cheredeprince.net/">Chère de Prince</a></strong>
            </div>`

marienfressinaud commented on pages/cloud-providers.html:
```diff
@@ -28,33 +28,18 @@
         <div class="card flow-small">
             <div class="card__title">
-                Chère de Prince
+                La Bécasse
```
I would keep "Chère de Prince" as "La Bécasse" is only the nickname of the person managing Chère de Prince (see https://cheredeprince.net "Qui suis-je ?")

vdbhb59 replied:
Actually, I got it from there only, but yeah, I agree. I will revert the same.

marienfressinaud commented on pages/cloud-providers.html:
```diff
@@ -28,33 +28,18 @@
-            </p>
-
-            <p>
-                <strong><a href="https://facil.services/">facil.services</a></strong>
```
FACiL still provides FreshRSS (see "[Journal FACiLe](https://journal.facil.services/)")

vdbhb59 replied:
Probably I searched in and out, but since I am translated using a translator, maybe I got confused. Will revert that change.

marienfressinaud commented on pages/cloud-providers.html:
```diff
@@ -110,39 +95,10 @@
-            <p>
-                <strong><a href="https://www.zaclys.com">zaclys.com</a></strong>
-            </p>
-        </div>
```
Zaclys still provides FreshRSS (see "[Flux](https://www.zaclys.com/flux/)", the screenshot is wrong though)

vdbhb59 replied:
Ahh. I removed it based on the changes and the "Flux" mentioned, which I believed to be Flux reader, and not exactly FreshRSS. Is there a instance page you can direct me to? I will add that as well, as I could not locate one myself.

marienfressinaud reviewed (changes requested):
~vern instance seems dead indeed, but the other providers still proposes FreshRSS as a service.

vdbhb59 commented:
All changes done as requested, and Landing pages are also available along with the main instance pages. Kindly check and merge.

vdbhb59 commented:
Unless anything pending for me, may I request this PR be merged?

Alkarex commented:
Ping @marienfressinaud

marienfressinaud commented:
I've simplified and made more coherent the links. I prefer to have the links at the same place in the cards (at the bottom), and I made the FreshRSS links a bit smaller.

I've also changed the layout of the cloud providers to grid instead of flexbox in order to decrease the risk of breaking the layout (it happened in your PR, but you're not the first).

How it looks now:

![The list of FreshRSS cloud providers. The links to the FreshRSS servers are next to the links to the cloud providers.](https://github.com/FreshRSS/freshrss.org/assets/1436309/7dcc8d42-aa7c-4299-a37a-fdf1dc81fd44)

vdbhb59 commented:
> I've simplified and made more coherent the links. I prefer to have the links at the same place in the cards (at the bottom), and I made the FreshRSS links a bit smaller.
> 
> I've also changed the layout of the cloud providers to grid instead of flexbox in order to decrease the risk of breaking the layout (it happened in your PR, but you're not the first).
> 
> How it looks now:
> 
> ![The list of FreshRSS cloud providers. The links to the FreshRSS servers are next to the links to the cloud providers.](https://user-images.githubusercontent.com/1436309/274055767-7dcc8d42-aa7c-4299-a37a-fdf1dc81fd44.png)

Thanks. Do I need to do anything further? Will you be merging this PR please.

marienfressinaud commented:
If it's fine to the others, we can merge! :)
